internal/loader/configuration_setting_loader.go

```diff
@@ -743,3 +759,51 @@ func MergeSecret(secret map[string]corev1.Secret, newSecret map[string]corev1.Se
 
 	return nil
 }
+
+func createCorrelationContextHeader(provider acpv1.AzureAppConfigurationProvider, clientManager ClientManager) http.Header {
```

Comment:
Can we also report `RequestType`, `Host`, and `Filter`?

https://github.com/Azure/AppConfiguration-DotnetProvider/blob/main/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs#L23

Reply:
@zhenlan  Added the `RequestType` and `Host`. In terms of the `Filter`, it's not cheap to report it from k8s provider, not trivial in K8s provider, two main reasons:

1. We currently don't parse the feature flags in K8s provider, we just get them and put into configMap, if we need to report the filters, we need to parse the feature flag values to know what filters are set. 
2. K8s provider manages all yaml resouces, we need to let the controller can remember/manage all filter usage information for each resource.

How do you think about it? We add the `Filter` for now, or add in the future on demand?

Reply:
If we are going to do dedup, we may parse it already. cc: @linglingye001

Reply:
We don't parse feature flag now, the implementation of feature flag deduplication is based on feature flag key.

Reply:
I think Zhenlan's view is that we need to dedup by id/name, then drives us having to parse it.